cmd/osmosisd/cmd/testnetify/testnetify.py

```diff
@@ -1,34 +1,49 @@
 import json
 import subprocess
 import re, shutil, tempfile
-import os
-from datetime import date
+from datetime import datetime
 
 
 #get values from your priv_validator_key.json to later switch with high power validator
 
 #get bas64
 result = subprocess.run(["osmosisd","tendermint","show-validator"], stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
 base64 = result.stdout.strip()
+##base64 = '{"@type":"/cosmos.crypto.ed25519.PubKey","key":"3QVAkiUIkKR3B6kkbd+QqzWDdcExoggbZV5fwH4jKDs="}'
+##bash64_wal = '{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A2MR6q+pOpLtdxh0tHHe2JrEY2KOcvRogtLxHDHzJvOh"}'
 
 #get validator cons pubkey
 val_pubkey = base64[base64.find('key":') +6 :-2]
+##val_pubkey = "3QVAkiUIkKR3B6kkbd+QqzWDdcExoggbZV5fwH4jKDs="
+#CAN MODIFY
+val_pubkey_wal = "A2MR6q+pOpLtdxh0tHHe2JrEY2KOcvRogtLxHDHzJvOh"

```

Comment:
What does wal mean here?

(wal in tendermint / databases means write-ahead-log)

Reply:
Oof I meant it as shorthand for wallet, will make sure that gets changed next iteration!